.env

```diff
@@ -18,7 +18,7 @@ CURRY_SUCCESS_LABEL="Signed CLA"
 PUBSUBHUBBUB_SECRET=YOUR_PUBSUBHUBBUB_SECRET
 NEW_RELIC_APP_NAME=Supermarket
 PUBSUBHUBBUB_CALLBACK_URL=http://example.com
-FEATURES=cla,join_ccla,tools,fieri,announcement,github
+FEATURES=cla,join_ccla,tools,fieri,announcement,github, collaborator_groups
```

Comment:
adds collaborator groups to the default features for use in development environments